.github/workflows/upload-android-test-specs.yml

```diff
@@ -45,50 +29,3 @@ jobs:
       models: stories110M
       devices: samsung_galaxy_s22
       delegates: xnnpack
```

Comment:
Now this validation step will run the `android-perf.yml` with the specialized test-spec correct?

Reply:
Yup, it will test the specialized test spec